_studio/mfx_lib/shared/src/mfx_h264_enc_common_hw.cpp

```diff
@@ -3417,8 +3417,16 @@ mfxStatus MfxHwH264Encode::CheckVideoParamQueryLike(
 
     if (extOpt2->LookAheadDS > MFX_LOOKAHEAD_DS_OFF)
     {
-        par.calcParam.widthLa  = mfx::align2_value((par.mfx.FrameInfo.Width  / LaDSenumToFactor(extOpt2->LookAheadDS)), 16);
-        par.calcParam.heightLa = mfx::align2_value((par.mfx.FrameInfo.Height / LaDSenumToFactor(extOpt2->LookAheadDS)), 16);
+        if (extOpt2->LookAheadDS > MFX_LOOKAHEAD_DS_4x)
+        {
+            extOpt2->LookAheadDS = MFX_LOOKAHEAD_DS_UNKNOWN;
+            changed = true;
```

Comment:
It's more looking as ```unsupported``` for me. I don't see this parameter changed below which is assumed when ```UNKNOWN``` is used. I think you need either to set it off or copy logic of assignment of some value from ```SetDefaults```